pkg/manager/member/tidbcluster_status_manager.go

```diff
@@ -34,7 +34,7 @@ const (
 	componentPrefix     = "/topology"
 	tidbPrefix          = "/topology/tidb"
 
-	tidbAddrPattern = `%s-tidb-\d+\.%s-tidb-peer\.%s\.svc%s\:\d+`
+	tidbAddrPattern = `%s-tidb-\d+\.%s-tidb-peer\.%s\.svc%s`
```

Comment:
Why remove `\d+` in the end? In this case, the addresses in the other Kubernetes clusters with the same name and namespace will also be matched and deleted, right?

Reply:
https://github.com/pingcap/tidb-operator/pull/3929/commits/d570f0f7f4e0f804fd66c08049aaeb91ca939856